jekyll/_cci2/server-3-install-build-services.adoc

```diff
@@ -380,6 +380,15 @@ Once you have configured the environmental variables and contexts, rerun the rea
 
 image::realitycheck-pipeline.png[Screenshot showing the realitycheck project building in the CircleCI app]
 
+==== Enabling Nomad Client for External Image Repository
+To enable the Nomad client so it can download images from an external or private repo, you can implement one of the two solutions suggested below:
+
+* Keep the image registry as `Public` within `Internal network` so the Nomad client can download the images without any authentication.
+* Or, Embed the private image repository credentials in the Nomad client image at `/root/.docker/config.json`, assuming the credentials are static.
```

Comment:
I think this point deserves more details:
- Are we telling the customer to publish their own AMI to use in the nomad cluster?
- what is this file (`/root/.docker/config.json`) supposed to look like, can we publish an example or a link to dockers documentation
- what if the credentials are not static?

Reply:
Ripped off this sections, as this is required customer to alter the nomad client image.